pkg/securitypolicy/securitypolicy.go

```diff
@@ -16,10 +16,17 @@ import (
 )
 
 //go:embed framework.rego
-var FrameworkCode string
+var frameworkCodeTemplate string
```

Comment:
maybe not in this PR, but it's worth to think about other people wanting to use this as a library to implement their own policies. How it is right now the package going towards being rego only, given we have all of these embeds here now, rather than in securitypolicyenforcer_rego.go.

Reply:
The reason for them being here is the build tag. As these values are needed by the simulator, they need to live here. If/when the build tag is removed, they could be moved to the `rego` file.